packages/server/graphql/types/AtlassianIntegration.ts

```diff
@@ -102,11 +107,8 @@ const AtlassianIntegration = new GraphQLObjectType<any, GQLContext>({
             'A list of projects to restrict the search to. format is cloudId:projectKey. If null, will search all'
         }
       },
-      resolve: async (
-        {teamId, userId, accessToken, cloudIds}: AtlassianAuth,
-        {first, queryString, isJQL, projectKeyFilters},
-        {authToken}: GQLContext
-      ) => {
+      resolve: async ({teamId, userId, accessToken, cloudIds}, args: any, {authToken}) => {
```

Comment:
I was wondering why you didn't do `args: IssueArgs` and remove the casting below, but I see it makes the `field` type incompatible. Annoying that you can't define the args type in a `GraphQLObjectType`